__tests__/db.test.js

```diff
@@ -35,6 +35,62 @@ describe("GET /api/topics", () =>{
     })
 })
 
+describe("GET /api/articles/:article_id/comments", () =>{
+    test("Returns 200 status code with an array", () =>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .then((res)=>{
+            expect(res.statusCode).toBe(200)
+            expect(Array.isArray(res.body))
+        })
+    })
+    test("Returns an array with appropriate data", ()=>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .then(({body})=>{
+            body.forEach((comment)=>{
+                expect(comment).toHaveProperty("comment_id", expect.any(Number))
+                expect(comment).toHaveProperty("votes", expect.any(Number))
+                expect(comment).toHaveProperty("created_at", expect.any(String))
+                expect(comment).toHaveProperty("author", expect.any(String))
+                expect(comment).toHaveProperty("body", expect.any(String))
+                expect(comment).toHaveProperty("article_id", expect.any(Number))
+            })
+        })
+    })
+    test("Comments are returned as most recent first", () =>{
+        return request(app).get("/api/articles/1/comments")
+        .then(({body})=>{
+            expect(new Date(body[0].created_at).getTime() > new Date(body[1].created_at).getTime()).toBe(true)
+            expect(new Date(body[1].created_at).getTime() > new Date(body[2].created_at).getTime()).toBe(true)
+            expect(new Date(body[2].created_at).getTime() > new Date(body[3].created_at).getTime()).toBe(true)
+            expect(new Date(body[3].created_at).getTime() > new Date(body[4].created_at).getTime()).toBe(true)
+            expect(new Date(body[4].created_at).getTime() > new Date(body[5].created_at).getTime()).toBe(true)
+        })
```

Comment:
^^ this is an interesting way to achieve this task, but it must have been fairly time consuming. Have a look at jest-sorted for the next time you have to check how something is sorted :)